server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -277,7 +277,10 @@ private ConfigProperties() {
 
     public static final String SWAGGER_ENABLED = "candlepin.swagger.enabled";
     public static final String TOKEN_ENABLED = "candlepin.token.enabled";
-
+    /**
+     * Keycloak Filepath needed for authentication
+     */
+    public static final String KEYCLOAK_FILEPATH = "/etc/candlepin/keycloak.json";
```

Comment:
The way this class works is the constant you're defining here is the config name that will appear in candlepin.conf. Like the above TOKEN_ENABLED, I would expect this to look like "candlepin.keycloak.config" or some such.

Once this is defined, we may also want to define a default, further down in the file (see how TOKEN_ENABLED is used for an example). However, whether or not we add a default depends on the feature in question. In this case, I'm not so sure.